src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/BaseTest.java

```diff
@@ -38,6 +45,22 @@
     protected static final String REFERENCE_NUMBER = "000MC001";
     protected final Claim claimAfterSavingWithResponse = SampleClaim.getWithDefaultResponse();
 
+    @ClassRule
+    public static PostgreSQLContainer postgres = new PostgreSQLContainer("postgres:9.6")
+        .withDatabaseName("claimstore");
+
+    public static class Initializer implements ApplicationContextInitializer<ConfigurableApplicationContext> {
```

Comment:
does this have to be public? can it just be static?

Reply:
I tried all other modifiers and only public seems to work. Private does not compile, remaining 2 fail at runtime.